src/Pages/PROFILE.js

```diff
@@ -1,8 +1,31 @@
+/* eslint-disable no-unreachable */
 import React from 'react';
 import { useSelector } from 'react-redux';
 import MyProfile from '../component/profile';
+import '../component/my-profile/profile.css';
 
 const Profile = () => {
+  const myRockets = useSelector((state) => state.rockets);
+  const myReserveRockets = myRockets.rocketsData.filter((rocket) => rocket.reserved);
+
+  const handleReservations = () => {
+    if (myReserveRockets.length) {
+      return myReserveRockets.map((rocket) => (
+        <p key={rocket.id} className="my-rocket">
+          {rocket.name}
+        </p>
+      ));
+    }
+    return <h2>No Reserved Rockets</h2>;
+  };
+  return (
+    <div className="profile">
+      <section className="my-rockets-sec">
+        <h3 className="my-heading"> My Rockets </h3>
+        <div className="my-rockets">{handleReservations()}</div>
+      </section>
+    </div>
+  );
```

Comment:
It seems that the reserved dragons don't show on the profile page, so I suggest putting the lines from 33 to 49 inside return () and changing the place of lines 29 and 30 to be under line 9. 👍🙂